### ISSUE TITLE
".card .card-content p"  should not have color set to "inherit"

### ISSUE THREAD

**Issue by IAMtheIAM** [Dogfalo/materialize#5305]
The following CSS in [_cards.scss](https://github.com/Dogfalo/materialize/blob/master/sass/components/_cards.scss#L147) makes no sense:

```css
.card .card-content p {
    margin: 0;
    color: inherit; // <--- this makes no sense
}
```

This prevents applying a class to a `p` element which would change the style. For example:

```css
.error-state {
    color: red;
}
```

```html
<div class="card">
    <div class="card-content">
        <p class="error-state">Error</p>
    </div>
</div>
```

The `red` color would be overridden by `inherit`, defaulting it to black or whatever the parent is.

## Proposed solution
remove `color:inherit` from the class specified above

**Comment by Dogfalo** [comment]
Fixed in 7b13e49

**Comment by IAMtheIAM** [comment]
Thx!
